src/main/kotlin/coffeebot/commands/CoffeeWager.kt

```diff
@@ -132,6 +124,24 @@ val adjudicate = Command("!adjudicate", "Adjudicate a bet. !adjudicate ID WINNER
     }
 }
 
+val pay = Command("!pay", "Register a coffee payment to another user") { message ->
+    val groups = payRegex.matchEntire(message.contents)?.groupValues
+    val payeeName = groups?.getOrNull(1)
+    val amountString = groups?.getOrNull(2)
+    if (payeeName == null || amountString == null) {
+        message.reply("Expecting !pay USER AMOUNT")
+        return@Command
+    }
+
+    val payerName = message.user.name
+    val amount = amountString.toInt()
```

Comment:
Sounds good. Do you also want to reuse the `(a|one|two|three|[1-7])(?: cups?(?: of)?)?(?: coffees?)?` syntax? I hadn't thought of it but it might be more consistent

Reply:
That seems like a good idea, especially if you share the regex in both places